tests/html/legend-Tests.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <title>Legend Tests</title>
+
+        <link rel="stylesheet" media="screen" href="../lib/infusion/lib/qunit/css/qunit.css" />
+
+        <script src="../lib/infusion/lib/qunit/js/qunit.js"></script>
+        <script src="../../src/lib/infusion/infusion-custom.js"></script>
+        <script src="../../src/lib/d3/d3.js"></script>
+        <script src="../lib/infusion/test-core/jqUnit/js/jqUnit.js"></script>
+        <script src="../lib/infusion/test-core/jqUnit/js/jqUnit-browser.js"></script>
+        <script src="../lib/infusion/test-core/utils/js/IoCTestUtils.js"></script>
+
+        <script src="../../src/js/utils.js"></script>
+        <script src="../../src/js/d3ViewComponent.js"></script>
+        <script src="../../src/js/legend.js"></script>
+        <script src="../js/legendTests.js"></script>
+        <style>
+        .gpii-ca-pieChart-table td.gpii-ca-pieChart-table td {
```

Comment:
Is the test dependent on this css? If not, it should be removed. In the case to add the component css to test files, the css should be in a css file in "src/css/" directory.